src/CoreMidi/MidiServices.cs

```diff
@@ -1456,6 +1467,16 @@ public MidiEntity GetEntity (nint entityIndex)
 			return new MidiEntity (h);
 		}
 
+		[NoiOS]
```

Comment:
hmm... missed that one before (multiple times it seems) :(
@mandel-macaque `[NoiOS]` won't remove the symbol from the binary (unless it's processed by the generator), which can be an issue